packages/amplify_datastore/example/lib/models/Blog.dart

```diff
@@ -137,6 +183,18 @@ class Blog extends Model {
         isRequired: false,
         ofModelName: (Post).toString(),
         associatedKey: Post.BLOG));
+
+    modelSchemaDefinition.addField(ModelFieldDefinition.field(
+        key: Blog.CREATEDAT,
```

Comment:
Related to my query field comment, `createdAt` and `updatedAt` should not have primary key.

Reply:
Our ModelFieldDefinition.field method is a bit strange - it uses the QueryField.fieldName property to get the name of the field being passed.  I need to update the method to be able to take the field name directly instead of needing a QueryField to be initialized. 

```
 static ModelFieldDefinition field(
      {required QueryField key,
      bool isRequired = true,
      bool isArray = false,
      ModelFieldType ofType = const ModelFieldType(ModelFieldTypeEnum.string),
      ModelAssociation? association,
      List<AuthRule>? authRules}) {
    return ModelFieldDefinition(
        name: key.fieldName,
```

Reply:
Same change needed, I'll need to talk to Aaron to update codegen here.

Reply:
QueryFields removed